Automattic/vip-go-mu-plugins
# Don't skip wpcom_vip_load_plugin for wp-activate.php

## Description

#4247 introduced a regression that resulted in client-mu-plugins not being loaded  in `wp-activate.php`.

This is because `wp-activate.php`  defines `WP_INSTALLING`.

This PR addresses the regression by explicitly checking whether current page is `/wp-activate.php` or not.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Check out the PR.
2. Add a dummy plugin to client-mu-plugins/ to make sure it's loaded during the further testing steps
3. On a multisite with open registration go to register page http://vip.vipdev.lndo.site/wp-login.php?action=register
4. Create a new account
5. In MailHog click on the link to `wp-activate.php`, e.g. http://vip.vipdev.lndo.site/wp-activate.php?key=1d7ace249259ff51 - make sure the dummy plugin is loaded
6. Also visit a link directly at /wp-activate.php and make sure the plugin is loaded.
7.
